lib/supply/uploader.rb

```diff
@@ -1,14 +1,15 @@
 module Supply
   class Uploader
     def perform_upload
-      FastlaneCore::PrintTable.print_values(config: Supply.config, hide_keys: [:issuer], title: "Summary for supply #{Supply::VERSION}")
+      # REVIEW: hide :json ? Is it sensitive enough?
+      FastlaneCore::PrintTable.print_values(config: Supply.config, hide_keys: [], title: "Summary for supply #{Supply::VERSION}")
```

Comment:
Yea, we shouldn't display the contents of the .json here, but the path to the file would be OK.